src/SFA.DAS.AdminService.Web/Validators/SearchViewModelValidator.cs

```diff
@@ -0,0 +1,103 @@
+﻿using FluentValidation;
+using SFA.DAS.AdminService.Web.Controllers;
+using SFA.DAS.AssessorService.Web.Infrastructure;
+using System;
+using System.Security.Cryptography;
+
+namespace SFA.DAS.AdminService.Web.Validators
+{
+    public class SearchInputViewModelValidator : AbstractValidator<SearchInputViewModel>
+    {
+        public SearchInputViewModelValidator()
+        {
+            When(vm => vm.SearchMode == SearchMode.Standards, () =>
+            {
+                RuleFor(vm => vm.SearchString)
+                    .NotEmpty()
+                    .WithMessage("Enter name or number")
+                    .Must(x => x == null || x.Trim().Length > 1)
+                    .WithMessage("Name or number must be at least 2 characters.");
+            });
+
+            When(vm => vm.SearchMode == SearchMode.Frameworks, () =>
+            {
+                RuleFor(vm => vm.FirstName)
+                    .NotEmpty()
+                    .WithMessage("Enter first name")
+                    .Must(x => x == null || x.Trim().Length > 1)
+                    .WithMessage("First name must be at least 2 characters.");
+
+                RuleFor(vm => vm.LastName)
+                    .NotEmpty()
+                    .WithMessage("Enter last name")
+                    .Must(x => x == null || x.Trim().Length > 1)
+                    .WithMessage("Last name must be at least 2 characters.");
+
+                RuleFor(vm => vm.Day).NotEmpty().WithMessage("The date of birth must include day").DependentRules(() =>
```

Comment:
There are other example of doing this in admin - usually you check for any empty day, month or year - then it's - enter a date
Then take the string "day/month/year" and try parse it to a date - fails it's an invalid date
Then if it's a date is it in the past